File: 1655-116-1051-height-checker/1655-116-1051-height-checker.py
class Solution(object):
    def heightChecker(self, heights):
        expected=sorted(heights)
        result=0
        for i in range(len(heights)):
            if expected[i]!=heights[i]:
                result+=1
        return result
       
        """
        :type heights: List[int]
        :rtype: int
        """
        

File: 1655-116-1051-height-checker/1655-116-1051-height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int[] expected=heights.clone();
        Arrays.sort(expected);
        int result=0;
        for(int i=0;i<heights.length;i++)
         {  if(heights[i]!=expected[i])
              result++;
          }
          return result;
    }
}